paper/paper.md

```diff
@@ -61,7 +61,7 @@ We also hope that by providing documented and performant implementations of publ
 ProbNumDiffEq.jl is also the only software package in Julia, at the time of writing, that provides a comprehensive set of probabilistic numerical ODE solvers.
 Outside of Julia, two other software packages provide a similar functionality.
 ProbNum [@wenger2021probnum]
-is a Python package that implements probabilistic numerical for various numerical problems, including linear systems, quadrature, and ODEs.
+is a Python package that implements probabilistic numerical solution methods for various numerical problems, including linear systems, quadrature, and ODEs.
```

Comment:
Thanks a lot for catching this! Would the following also work for you? "Probabilistic numerical methods" is a term that appears quite often in the literature.
```suggestion
is a Python package that implements probabilistic numerical methods for various numerical problems, including linear systems, quadrature, and ODEs.
```

Reply:
That's okay